cafebabel/django_migrations.py

```diff
@@ -1,43 +1,174 @@
 import json
-from datetime import datetime, timezone
+from datetime import datetime, timedelta, timezone
 
 import click
-from cafebabel.users.models import Role
+from cafebabel.articles.models import Article
+from cafebabel.articles.tags.models import Tag
+from cafebabel.articles.translations.models import Translation
+from cafebabel.users.models import Role, User
+from mongoengine import errors as mongo_errors
 from progressist import ProgressBar
 
+PROGRESSIST_TEMPLATE = ('{prefix} {animation} {percent} ({done}/{total}) '
+                        'ETA: {eta:%H:%M} {elapsed}')
+NOW = datetime.now()
```

Comment:
Please remind me why not storing `utcnow()` instead?

Reply:
That date is not stored, just compared to exclude 3-months old content. I guess we are not about an hour precision 😉